### ISSUE TITLE
Text drag'n'drop on editor selections

### ISSUE THREAD

**Issue by StevenRCE0** [zed-industries/zed#4958]
### Check for existing issues

- [X] Completed

### Describe the feature

Though it seems "un-geeky" to use mouse to rearrange blocks of code, it would be nice to have! 

### If applicable, add mockups / screenshots to help present your vision of the feature

![Screen Recording 2023-03-13 at 14 01 51-0001](https://user-images.githubusercontent.com/40051361/224620308-0bcc5b4e-4b38-4fb1-b787-7daf37f69f4e.png)

**Comment by chadananda** [comment]
This was pointed out over a year ago. How do basic editor UI features get left out? 

This is why VS-Code wins, their team first implemented feature-complete editing UI.

**Comment by fullyDeepak** [comment]
Any update on this ?

**Comment by lcdss** [comment]
I miss this feature from vscode, but it seems it's close to be released. Thanks to the contributors.

**Comment by meliborn** [comment]
Was it released? Should I enable something?

**Comment by smitbarmase** [comment]
It's enabled by default. Just hold on to selection for more than 300ms, and drag.

**Comment by injust** [comment]
> Dragging text from another application into a Zed editor should also work.

@smitbarmase Would you prefer to open a new discussion for this?

**Comment by smitbarmase** [comment]
Created one here: https://github.com/zed-industries/zed/discussions/34930

**Comment by digitalextremist** [comment]
Seems like this functionality disappeared? `0.196.5` has the setting now, but the functionality is gone.

**Comment by WantenMN** [comment]
It should also support dragging from one pane to another pane.

**Comment by digitalextremist** [comment]
Has anyone noticed this functionality gone, even with this setting in place?

```
  "drag_and_drop_selection": { "enabled": true, "delay": 300 }
```

Currently on `0.199.9` and about to update

---

Issue persists at `0.200.5` [ estimated to have seen this for >20 patch revisions and ~2 minor versions ]

**Comment by eandersons** [comment]
@digitalextremist, did you try `"delay": 0`? From what I have observed, drag and drop with delay can be quite tricky.

**Comment by digitalextremist** [comment]
Yes, I did try `delay: 0` @eandersons; unfortunately, no difference!

**Comment by eandersons** [comment]
@digitalextremist, that is unfortonate to hear.

What is the operating system on the computer you are experiencing drag and drop not working?

Do you have a chance to check if the issue can be observed on other machines?

Either way I think that you should create a new issue regarding the problems you are experiencing if that has not been already done, as this one is specifically about adding drag and drop functionality, which has been done.

**Comment by digitalextremist** [comment]
Did ticket this, here, @eandersons: https://github.com/zed-industries/zed/issues/36968

@smitbarmase seems to think it might be isolated to `Linux Mint`

Have not been able to check with a different machine, but will try to also!
